spotinst/mrscaler_aws_instance_groups/fields_spotinst_mrscaler_aws_instance_groups_core.go

```diff
@@ -50,6 +56,11 @@ func SetupCoreGroup(fieldsMap map[commons.FieldName]*commons.GenericField) {
 			return nil
 		},
 		func(resourceObject interface{}, resourceData *schema.ResourceData, meta interface{}) error {
+			if strategy, ok := resourceData.GetOk(string(mrscaler_aws_strategy.Strategy)); ok && strategy != "" {
+				if strategy == mrscaler_aws_strategy.Wrap {
+					return nil
+				}
+			}
```

Comment:
You can simplify the code by combining the two `if` statements:

```suggestion
			if strategy, ok := resourceData.GetOk(string(mrscaler_aws_strategy.Strategy)); ok && strategy == mrscaler_aws_strategy.Wrap {
				return nil
			}
```